include/seqan3/alphabet/gap/gap.hpp

```diff
@@ -117,32 +117,32 @@ struct gap
 
     //!\name Comparison operators
     //!\{
-    constexpr bool operator==(gap const &) const noexcept
+    friend constexpr bool operator==(gap const &, gap const &) noexcept
```

Comment:
remove reference

Reply:
This doesn't make a difference, because there is no parameter anyway. I assume this function will always be replaced with just the return value in the code.

Reply:
(We haven't discussed this for SeqAn3, because luckily we don't have so many instances of "dead" parameters, but I think `foo const &` in this case makes it even easier to see that there is no parameter name than just `foo`)